### ISSUE TITLE
Uncaught ReferenceError: _Buffer is not defined

### ISSUE THREAD

**Issue by w-ah** [williamkapke/bson-objectid#50]
**Description**
_Buffer is not defined in Node.js. If the program reaches this statement, an error will be thrown and, if uncaught, will crash the node process. 

**Minimal Reproduction**
In Node.js REPL (tested against v16.13.2):
```bash
> _Buffer
Uncaught ReferenceError: _Buffer is not defined
```

**Workaround**
For Node.js, alias the Buffer global as _Buffer in your program entrypoint so that it is defined before calling anything in bson-objectid:
```js
global['_Buffer'] = Buffer;
```

**Possible Fix**
Attempt to use _Buffer, else fall back to using Buffer. Just using normal Buffer global fixes things for Node.js so a global alias is not required.

**Comment by w-ah** [comment]
Not actually sure why _Buffer is being used anyway?

**Comment by titanism** [comment]
This is fixed and released in v2.0.4 of `bson-objectid` to npm.  Until we have access to `master` branch of this repository we have published our changes in a fork at https://github.com/cabinjs/bson-objectid.
